corehq/apps/api/resources/v0_5.py

```diff
@@ -860,15 +859,18 @@ def dispatch_list(self, request, **kwargs):
                 raise
 
     def obj_get_list(self, bundle, **kwargs):
-        return self.get_object_list(bundle.request)
+        feature_flag = kwargs.get("feature_flag")
+        return self.get_object_list(bundle.request, feature_flag=feature_flag)
 
-    def get_object_list(self, request):
+    def get_object_list(self, request, feature_flag=None):
         couch_user = CouchUser.from_django_user(request.user)
         results = []
         for domain in couch_user.get_domains():
             if not domain_has_privilege(domain, privileges.ZAPIER_INTEGRATION):
                 continue
             domain_object = Domain.get_by_name(domain)
+            if feature_flag and feature_flag not in toggles.toggles_enabled_for_domain(domain):
```

Comment:
Can do that, my thought was that checking 'flag in []' would implicitly handle the wrong flag since no domain's enabled flags will match it.
 Alternatively, if we should throw error then can make an explicit check for it. What do you suggest?

Reply:
Returning a 400 Bad Request would be best practice. Not doing so also makes it impossible to tell if you got the name wrong or if the domain doesn't have the flag enabled.

Since these names are just python variable names too it's possible that someone could rename the toggle in python and break the integration.